tests/test_metrics.py

```diff
@@ -0,0 +1,106 @@
+"""Test metrics module."""
+
+import numpy as np
+
+
+def test_pixel_error():
```

Comment:
I know `pixel_error()` is super simple but it is the same as `RegressionRMSE` no? Can we just use the latter you think?

Reply:
let's revisit later, there's some reshaping that would need to happen that would actually obscure what is going on in this function; since we only use it for logging and not gradients i think it's fine to keep separate for now, but I understand the desire to unify